### PR TITLE
Added footer links and form for beta signup.

### PR DESCRIPTION
**Pull request by lilibobear**
Added footer links and Hubspot signup.

### CODE REVIEW

File: www/src/pages/privacy.mdx
Comment (jamilbk) [inline www/src/pages/privacy.mdx:1]: Probably want to rename this file to `_privacy.mdx` to mark it as draft

File: www/src/pages/terms.mdx
Comment (jamilbk) [inline www/src/pages/terms.mdx:1]: Probably want to add a leading underscore `_terms.mdx` so this page isn't published until it's ready

File: www/src/pages/self-hosted-firezone.mdx [hunk 1]
```diff
@@ -0,0 +1,391 @@
+---
+title: Firezone self-hosted
+description: Deploy Firezone self-hosted.
```
Comment (jamilbk) [inline www/src/pages/self-hosted-firezone.mdx]:
```suggestion
description: Deploy a self-hosted instance of Firezone.
```

File: www/src/pages/index.mdx [hunk 2]
```diff
@@ -391,14 +395,117 @@ description: Deploy in minutes to secure remote access to your private resources
...
+       </center>
+      </div>
+      <div className="col col--6">
+        <center>
+       <a className="button button--primary" href="/beta/signup">
+        Join the beta waitlist
+       </a>
+       </center>
+      </div>
+    </div>
```
Comment (jamilbk) [inline www/src/pages/index.mdx]: Indentation

```suggestion
<div className="row margin-top--xl">
  <div className="col col--6">
    <center>
   <a className="button button--primary" href="/docs/deploy">
    Deploy now
   </a>
   </center>
  </div>
  <div className="col col--6">
    <center>
   <a className="button button--primary" href="/beta/signup">
    Join the beta waitlist
   </a>
   </center>
  </div>
</div>
```

File: www/src/pages/index.mdx
Comment (jamilbk) [inline www/src/pages/index.mdx:1]: Indentation in this file needs to be fixed. See https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

File: www/src/pages/beta/signup.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Firezone • Open Source Remote Access
+description: Firezone • Beta Signup
+---
+
+<div className="hero shadow--lw">
+  <div className="container">
+    <h1 className="hero__title">Firezone 1.0</h1>
+
```
Comment (jamilbk) [inline www/src/pages/beta/signup.mdx]: Remove empty line
```suggestion
```

File: www/docusaurus.config.js [hunk 4]
```diff
@@ -94,6 +94,14 @@ const config = {
...
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      announcementBar: {
+        id: 'support_us',
+        content:
+          'Firezone 1.0 is coming! <a target="_blank" rel="noopener noreferrer" href="/beta/signup">Sign up for early access.</a>',
+        backgroundColor: '#fafbfc',
+        textColor: '#091E42',
+        isCloseable: false,
+      },
```
Comment (jamilbk) [inline www/docusaurus.config.js]: Comment out until the form is ready
```suggestion
      /*announcementBar: {
        id: 'support_us',
        content:
          'Firezone 1.0 is coming! <a target="_blank" rel="noopener noreferrer" href="/beta/signup">Sign up for early access.</a>',
        backgroundColor: '#fafbfc',
        textColor: '#091E42',
        isCloseable: false,
      },*/
```

File: www/docusaurus.config.js [hunk 5]
```diff
@@ -153,6 +166,22 @@ const config = {
...
                 href: "/",
               },
+                {
+                  label: "Jobs (We're hiring!)",
+                  href: "https://www.ycombinator.com/companies/firezone",
+                },
+                // {
+                  // label: "Beta",
+                  // href: "/beta/signup",
+                // },
```
Comment (jamilbk) [inline www/docusaurus.config.js]:
```suggestion
                // {
                  // label: "1.0",
                  // href: "/1.0/signup",
                // },
```

File: www/docusaurus.config.js [hunk 6]
```diff
@@ -122,6 +130,11 @@ const config = {
             position: "left",
             "aria-label": "Contact support",
           },
+          // {
+           // href: "/beta/signup",
+           // label: "Beta",
+          // position: "left",
+          //},
```
Comment (jamilbk) [inline www/docusaurus.config.js]:
```suggestion
          // {
           // href: "/1.0/signup",
           // label: "1.0",
          // position: "left",
          //},
```

**Review by jamilbk — CHANGES_REQUESTED** [review]
Added some suggestions

File: www/src/pages/privacy.mdx
Comment (lilibobear) [reply www/src/pages/privacy.mdx:1]: agreed, I will do that in the future for draft pages!

File: www/src/pages/terms.mdx
Comment (lilibobear) [reply www/src/pages/terms.mdx:1]: Agreed

**Pull request merged**